docs/developing/troubleshooting.md

```diff
@@ -35,16 +40,20 @@ very likely you are seeing this error.
 If possible, configure the app to use a base URL of `''`, literally
 the empty string. Then it will send HTTP redirects without
 specifying a base URL. If that isn't possible, Sandstorm apps should
-look at the
+look at the `Host:` header for a base URL.
```

Comment:
Haha, ok! I'll keep it in mind, though I don't think I can verify it myself yet. I have another couple levels of understanding to go through first.

Don't feel any pressure to merge this if it would be better to close it and make a more correct PR. That's fine too.